pan3d/__main__.py

```diff
@@ -0,0 +1,18 @@
+from argparse import ArgumentParser
+from dataset_builder import DatasetBuilder
+
+parser = ArgumentParser(
+    prog="Pan3D",
+    description="Launch the Pan3D Dataset Builder",
+)
+
+parser.add_argument("-b", "--config_path")
+parser.add_argument("-d", "--dataset_path")
+
+args = parser.parse_args()
+
+builder = DatasetBuilder(dataset_path=args.dataset_path)
+if args.config_path:
+    builder.import_config(args.config_path)
+viewer = builder.viewer
+builder.server.start()
```

Comment:
I would replace 

```python
viewer = builder.viewer
builder.server.start()
```

by 

```python
builder.viewer.server.start()
```

Reply:
In some way, I would make builder not really "trame server" dependent. But we can work on that piece later on especially since it will be trivial.

Reply:
All the refactoring your are doing now, will make sure a change a piece of cake down the road.

Reply:
Good idea, changed in [0a844e2](https://github.com/Kitware/pan3d/pull/28/commits/0a844e2c9791ffe7cf23a5b37e165d91ba1bf8b4)